### ISSUE TITLE
β-VAE: LEARNING BASIC VISUAL CONCEPTS WITH A CONSTRAINED VARIATIONAL FRAMEWORK

### ISSUE THREAD

**Issue by nagataka** [nagataka/Read-a-Paper#42]
# Summary

#### Link

- Paper: [beta-VAE: Learning Basic Visual Concepts with a Constrained Variational Framework](https://openreview.net/forum?id=Sy2fzU9gl)
- Dataset: [dsprites-dataset](https://github.com/deepmind/dsprites-dataset)

## What is this

Introduce β-VAE, a new state-of-the-art framework for automated discovery of interpretable factorized latent representations from raw image data in a completely unsupervised manner

Main contributions are the following: 

1) we propose β-VAE, a new unsupervised approach for learning disentangled representations of independent visual data generative factors; 
2) we devise a protocol to quantitatively compare the degree of disentanglement learnt by different models; 
3) we demonstrate both qualitatively and quantitatively that our β-VAE approach achieves state-of-the-art disentanglement performance compared to various baselines on a variety of complex datasets

## Comparison with previous researches. What are the novelties/good points?

introduce an adjustable hyperparameter β that balances latent channel capacity and independence constraints with reconstruction accuracy

Unlike InfoGAN, β-VAE is stable to train, makes few assumptions about the data and relies on tuning a single hyperparameter β, which can be directly optimised through a hyperparameter search using weakly labelled data or through heuristic visual inspection for purely unsupervised data

## Key points

"With β > 1 the model is pushed to learn a more efficient latent representation of the data, which is disentangled if the data contains at least some underlying factors of variation that are independent."

Assuming V and W which are ground truth data generative factors (See chapter 2): conditionally independent factors v and conditionally dependent factors w
data generation is governed by these factors as p(x|v,w) and it is called Sim(v,w), where 'Sim' comes from "the true world Simulator"

Relationship with a set of generative latent factors z: p(x|z) approximates p(x|v,w)

Learn z with a posterior q(z|x)

the prior distribution is an isotropic unit Gaussian (p(z) = N(0,I))

#### Disentanglement Metric

A linear classifier based approach.
Fix one of the generative factors and randomly sampling all others. The goal of the classifier is to predict the index y of the generative factor that was kept fixed.

![スクリーンショット 2021-12-09 19 08 20](https://user-images.githubusercontent.com/2190811/145510732-5fa26ad8-9626-44ab-9e7b-b54b408fafc8.png)

## How the author proved effectiveness of the proposal?

- Qualitative evaluation on CelebA, chairs and faces
- Quantitative evaluation and compared with ICP, PCA, VAE, DC-IGN, and InfoGAN

## What should I read next?

- [Understanding disentangling in β-VAE](https://arxiv.org/abs/1804.03599)
